sdk/Trace/SimpleSpanProcessor.php

```diff
@@ -23,24 +28,38 @@ public function __construct(Exporter $exporter)
      */
     public function onStart(API\Span $span): void
     {
-        // nothing to do here
     }
 
     /**
      * @inheritDoc
      */
     public function onEnd(API\Span $span): void
     {
-        // @todo only spans with SampleFlag === true should be exported according to
-        // https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md#sampling
-        $this->exporter->export([$span]);
+        if ($this->running) {
+            // @todo only spans with SampleFlag === true should be exported according to
```

Comment:
I considered tackling this in this same PR but I figured *yet another* logical change in the same diff should be avoided if possible.

Reply:
We should probably create a bite sized issue to tackle this.